Sources/JavaScriptKit/JSValue.swift

```diff
@@ -138,3 +153,37 @@ public func setJSValue(this: JSObject, index: Int32, value: JSValue) {
                        rawValue.payload1, rawValue.payload2, rawValue.payload3)
     }
 }
+
+extension JSValue {
+    public func isInstanceOf(_ constructor: JSFunction) -> Bool {
```

Comment:
Is a doc comment for this available in some protocol that has `isInstanceOf` requirement (I don't remember if there is one off the top of my head)? If not, then it needs a doc comment as a `public` declaration.

Reply:
Nope, just forgot to document it.